courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2007 @@
+{
```

Comment:
<p>I'd remove the spacing above and below the <code>df_to_dataset	</code> function here. This typical pep8 formatting is not needed for notebook.</p><p> </p><p>I'd remove any extra spacing above code in the cells throughout the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Removed extra spacing above code in the cells throughout the notebook.</p><p> </p>